Tools/Scripts/webkitdirs.pm

```diff
@@ -1282,7 +1286,16 @@ sub XcodeOptions
 
 sub XcodeOptionString
 {
-    return join " ", XcodeOptions();
+    my @options;
+    foreach my $option (XcodeOptions()) {
+        my ($key, $value) = split('=', $option);
+        if ($key eq "OTHER_CFLAGS") {
+            push @options, "$key=\'$value\'";
+        } else {
+            push @options, $option;
+        }
+    }
+    return join " ", @options;
```

Comment:
The goal is to handle spaces in option arguments, yes? How about:
```suggestion
    return join " ", map { /\s/ ? "\"$_\"" : $_ } XcodeOptions()
```

Reply:
Done. Thanks.